src/apps/chifra/pkg/output/json.go

```diff
@@ -20,29 +20,44 @@ import (
 	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/validate"
 )
 
-// PrintJson marshals its arguments and prints JSON in a standardized format
-func PrintJson(serializable interface{}) error {
-	var response map[string]interface{}
+type JsonFormatted struct {
```

Comment:
I'm still a little bit behind you on how the Outputting stuff should work. We can talk about it on Monday.

Reply:
Sure. I just wanted to have this code in one place, so we always get consistent format